### PR TITLE
✨Add PE Version to Solution's Title

### PR DESCRIPTION
**Pull request by alexanderkasten**
**Changes:**

1. Get PE Version
2. Add PE Version to SolutionEntry Interface
3. Add PE Version to SolutionEntry
4. Show Version of PE on Solutions Title

preview

PR: #1389 

## How can others test the changes?

- checkout the branch
- `npm run electron-start-dev`
- check the screenshot 

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

### CODE REVIEW

File: src/modules/solution-explorer/solution-explorer-list/solution-explorer-list.ts [hunk 1]
```diff
@@ -126,8 +126,14 @@ export class SolutionExplorerList {
     const uriIsRemote: boolean = uri.startsWith('http');
 
     let solutionExplorer: ISolutionExplorerService;
+    let processEngineVersion: string;
     if (uriIsRemote) {
       solutionExplorer = await this._solutionExplorerServiceFactory.newManagementApiSolutionExplorer();
+
+      const response: Response = await fetch(uri);
+      const responseJSON: object & {version: string} = await response.json();
```
Comment (Vyperus) [inline src/modules/solution-explorer/solution-explorer-list/solution-explorer-list.ts]:
```suggestion
      const responseJSON: Object & {version: string} = await response.json();
```

Comment (alexanderkasten) [reply src/modules/solution-explorer/solution-explorer-list/solution-explorer-list.ts]: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Comment (alexanderkasten) [reply src/modules/solution-explorer/solution-explorer-list/solution-explorer-list.ts]: @ElRaptorus @Vyperus

**Comment by Vyperus** [comment]
LGTM, please fix the Jenkins build.

**Pull request merged**